src/index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--sun"></div>
```

Comment:
please add `calendar month length modifier`

Reply:
sorry, how can i do it ?
i have a var for this parametr in scss

Reply:
![image](https://user-images.githubusercontent.com/51528776/183162666-c44e9146-38fc-4243-b7c0-a4a9216b5168.png)
![image](https://user-images.githubusercontent.com/51528776/183163021-b290ed23-7b6f-4e9d-8c64-95335f1ad1eb.png)
![image](https://user-images.githubusercontent.com/51528776/183163240-6d8081b9-e0a2-4c5e-a952-a2f1a7decf88.png)